src/gax/src/exponential_backoff.rs

```diff
@@ -20,10 +20,22 @@
 //! [BackoffPolicy]: crate::backoff_policy::BackoffPolicy
 //! [PollingBackoffPolicy]: crate::polling_backoff_policy::PollingBackoffPolicy
 
-use crate::Result;
-use crate::error::Error;
 use std::time::Duration;
 
+/// The error type for exponential backoff creation.
+#[derive(thiserror::Error, Debug)]
+#[non_exhaustive]
+pub enum Error {
+    #[error("the scaling value ({value}) should be >= 1.0")]
+    Scaling { value: f64 },
+    #[error("the initial delay ({value:?}) should be greater than zero")]
+    InitialDelay { value: Duration },
+    #[error(
+        "the maximum delay ({value:?}) should be greater or equal than the initial delay ({initial:?})"
+    )]
+    MaximumDelay { value: Duration, initial: Duration },
```

Comment:
This is as much an initial delay error as it is a maximum delay error.

How about:
```rs
EmptyRange {
  initial: Duration,
  maximum: Duration,
}
```